src/lib/theme/_default-variables.scss

```diff
@@ -3,11 +3,11 @@
 */
 
 // Light theme
-$td-accent: mat-palette($td-teal, 900, 100, 900);
+$td-accent: mat-palette($td-teal, 900, 200, 900);
 $td-primary: mat-palette($td-slate, 500, 200, 800);
 $td-warn: mat-palette($mat-red, 900);
 
 // Dark theme
-$td-dark-accent: mat-palette($td-teal, 300, 900, 100);
+$td-dark-accent: mat-palette($td-teal, 300, 1200, 100);
```

Comment:
I always debate this.. why are lighter and darker switched here?

mat-palette uses `normal, lighter, darker`.. should it be something like this?

```
mat-palette($td-teal, 300, 100, 1200);
```

The reason i bring it up is because i was fighting this over the weekend.. darker was not being used properly in dark mode

Reply:
I forget, I think it has to do with how they're used by the Angular Material components in forgrounds vs. background colors. What was behaving wrong?

Reply:
OK I remember. It's because when applied to a light vs a dark background, the one with the higher contrast is reversed. So on light, darker is higher contrast against your background, and vice versa. Best way to see it is on the progress bar demos. Where else are you seeing it applied?

Reply:
When we create components, we typically try to get a hue, and i was trying to get darker from the theme.. and it was looking super bad.

In covalent, we cant assume that we need to switch `lighter` vs `darker` depending on dark vs light.. it needs to be consistent.

Reply:
Yeah we can certainly switch it back, happy to do it in this pr. Will do another to provide an override for the progress bar and then adjust others as we come across them. 

This is likely a consequence of there being no dark theme by default in angular material, so they effectively hard-coded light theme assumptions into their color choices/naming. Easy enough to handle another way.